.travis.yml

```diff
@@ -1,21 +1,26 @@
 sudo: required
 services:
   - docker
+  - mysql
```

Comment:
I use Travis' default service for mysql instead of defining one in docker-compose.test.yml, but I could change this if you want.

Reply:
Can we do something similar for rvm as well?
A majority of the build time is spent on compiling ruby.
If rvm is defined outside the docker environment it can be cached on Travis. 
We can also cache bundler.
It will greatly improve build time

Reply:
Hmm, I know what you're saying @kaustubh-nair. Actually I have faced this same issue before in Falko project. I'll describe what might be happening here:

Travis correctly identifies our repository as a ruby/rails project. So in order to make things, I'd say easier, it pre-installs our ruby version defined in `Gemfile` and pre-bundles our defined gems too. That's no problem so far, but the thing is that we don't need either rvm or even ruby installed in Travis runners, we only need docker. Since we use a base image provided by ruby org itself, we only need docker service working properly in our runners. I have faced similar in https://github.com/falko-org/Falko-API/issues/240. So just by adding those directives to our travis yml, it will quit pre-installing ruby and pre-bundling our gems, thus reducing our build time.

Reply:
Also, another way for us to reduce our build time is to accomplish what I have suggested in #604. Every travis build time we rebuild our docker image, if we version control our images there would be no need of that, also we could manage our docker images better.